worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageDriverImpl.java

```diff
@@ -486,36 +572,66 @@ private ZonedDateTime getStartDateTime(Job job) {
                 1, 0, 0, 0, 0, AB2D_ZONE);
     }
 
-    private void checkCoveragePeriodValidity(Job job, CoveragePeriod period) {
+    void checkCoveragePeriodValidity(Job job, CoveragePeriod period) {
         if (period.getStatus() == JobStatus.FAILED &&
                 period.getModified().isAfter(job.getCreatedAt())) {
             throw new CoverageDriverException("attempts to pull coverage information failed too many times, " +
                     "cannot pull coverage");
         }
     }
 
+    /**
+     * Page through coverage using a provided paging request
```

Comment:
When you say "page", it wasn't obvious whether you meant in the database or BFD.